src/_shared/types.ts

```diff
@@ -722,8 +722,8 @@ export enum EAssets {
 export enum EPostDetailsTab {
 	DESCRIPTION = 'description',
 	TIMELINE = 'timeline',
-	ONCHAIN_INFO = 'onchain info',
-	POST_ANALYTICS = 'post_analytics'
+	ONCHAIN_INFO = 'onchainInfo',
```

Comment:
this should be in snake case, please look at the rest of the file for code consistency